fs/fsshrink.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2018 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+
+import os
+import shutil
+
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+
+
+def clear_dmesg():
```

Comment:
@PraveenPenguin I am fine with this test having `clear_dmesg()` and `collect_dmesg()` now, but can you please work on having an avocado utility to have these functions, to start with.

This will benefit a lot of tests.

Reply:
@narasimhan-v sure make sense